app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -76,44 +72,47 @@ def phone=(phone)
 
   def stale_encrypted_phone?
     return false unless phone.present?
-    @_encrypted[:phone].stale?
+    encrypted_attributes[:phone].stale?
   end
 
   private
 
+  def encrypted_attributes
+    @_encrypted_attributes ||= {}
+  end
+
   def get_encrypted_attribute(name:, default:)
     getter = encrypted_attribute_name(name)
     encrypted_string = self[getter]
     return default unless encrypted_string.present?
-    build_encrypted_attribute(name, encrypted_string)
-    @_encrypted[name].decrypted
+    encrypted_attribute = encrypted_attributes[name]
+    return encrypted_attribute.decrypted if encrypted_attribute.present?
+    build_encrypted_attribute(name, encrypted_string).decrypted
   end
 
   def build_encrypted_attribute(name, encrypted_string)
-    @_encrypted ||= {}
-    @_encrypted[name] = EncryptedAttribute.new(
+    encrypted_attribute = EncryptedAttribute.new(
       encrypted_string,
       cost: attribute_cost,
       user_access_key: attribute_user_access_key
     )
-    self.attribute_user_access_key ||= @_encrypted[name].user_access_key
+    self.attribute_user_access_key ||= encrypted_attribute.user_access_key
+    encrypted_attributes[name] = encrypted_attribute
   end
 
   def set_encrypted_attribute(name:, value:, default:)
-    @_encrypted ||= {}
     setter = encrypted_attribute_name(name)
     new_value = default
     if value.present?
-      build_encrypted_attribute_from_plain(name, value)
-      new_value = @_encrypted[name].encrypted
+      new_value = build_encrypted_attribute_from_plain(name, value).encrypted
     end
     self[setter] = new_value
   end
 
   def build_encrypted_attribute_from_plain(name, plain_value)
     self.attribute_user_access_key ||= new_attribute_user_access_key
-    @_encrypted[name] = EncryptedAttribute.new_from_decrypted(
-      plain_value,
+    encrypted_attributes[name] = EncryptedAttribute.new_from_decrypted(
+      plain_value.downcase.strip,
```

Comment:
Should we add a test that verifies that the downcasing and stripping of attributes?